rails_app/app/helpers/currencies_helper.rb

```diff
@@ -44,7 +44,7 @@ def relativize_record_ex_rate(record)
 
     # if the user has a favorite currency get the record for the same date and relativize
     if user_signed_in? && current_user.favorite_currency != nil
-      ex_rate = CurrencyRecord.find_by(code: current_user.favorite_currency.code,
+      ex_rate = CurrencyRecord.find_by(currency_id: current_user.favorite_currency.code,
```

Comment:
Finding by ID but serving the currency code. Is this working as expected?

Reply:
Probably not, fixing it.